scopes/react/ui/compositions-app/composition-app.docs.mdx

```diff
@@ -0,0 +1,3 @@
+This mounts compositions into the DOM in the component preview.
```

Comment:
why don't you have here the metadata section with labels and everything?

Reply:
this is from Uri's changes - the rebase should remove this from the PR